### ISSUE TITLE
Migrate and organize Knack wikis to Gitbook

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#1196]
- Repo/Wiki owners identify which of their wiki pages should be migrated
- Other tasks will be identified

**Comment by amenity** [comment]
@dianamartin - I think this is done, yeah?

**Comment by dianamartin** [comment]
@amenity no, we have a lot of other repo wikis that haven’t made it over yet.
